config/database.yml

```diff
@@ -5,13 +5,17 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
-  <<: *default
-  database: db/development.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: todo-app
+  username: postgres
+  password: admin
```

Comment:
@ValeriaMejiaCortes including hardcoded passwords in the config files is a no-no. Please get the values from environment variables such as:

```
  username: <%= ENV['POSTGRES_USERNAME'] %>
  password: <%= ENV['POSTGRES_PASSWORD'] %>
```

Also, as requested in the card description, please update the README with the instructions other people need to follow in order to install PostgreSQL, create the database user, and anything else required to get the database up & running with the project.